INSTALL.pipeline

```diff
@@ -45,6 +45,23 @@ him know a username you'd like for yourself, if you don't already have one.
 He will set things up so your new pipeline server can coordinate with the 
 others, and will be allowed to upload finished WARCs to the Internet Archive.
 
+** STEP 2.5: PYTHON VENV **
```

Comment:
It isn't just a venv; it's a complete Python installation. I'd just call this section 'PYTHON'.